webpack.renderer.config.js

```diff
@@ -36,9 +36,10 @@ module.exports = {
       "@models": srcPaths("app/models"),
       "@renderer": srcPaths("app/renderer"),
```

Comment:
- why this one has path `"app/renderer"` and `@components` alias has path `./app/renderer/components` - I mean one of them has preceding dot while other isn't, are both of them working?
- can you please add some usage examples of aliases to our existing sample components (e.g. Heading), something like
```js
import { Heading } from "@components"
```

Reply:
When I decided which one to add, I test all of them srcPath("app"), path.resolve("app"), path.resolve(_dirname, "app") and all of them is working (I added some scss file to another directory, created webpack alies and added used that alias), but it is not workings in some places. Maybe you should add file format extension, as i added below aliases, but it works.

1) "@models": srcPaths("app/models"), "@renderer": srcPaths("app/renderer") this two was in the articles about boilerplate, I thought it is necessery) 
2) import { Heading } '@components/Heading'